src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -78,6 +78,28 @@ def get_dsn_content(self, dataset_name):
         response_json = self.request_handler.perform_request("GET", custom_args)
         return response_json
 
+    def create_uss_file(self, file_path, type, mode = 'rwxr-xr-x'):
+        """
+        Add a file or directory
+        Parameters
+        ----------
+        file_name of the file to add
+        type = "file"
+        mode Ex:- rwxr-xr-x
+
+        """
+
+        data = {
+            "type": type,
+            "mode": mode
+        }
+        
+        custom_args = self.create_custom_request_arguments()
+        custom_args["params"] = data
+        custom_args["url"] = "{}fs/{}".format(self.request_endpoint, file_path.lstrip("/"))
+        response_json = self.request_handler.perform_request("DELETE", custom_args, expected_code=201)
```

Comment:
Please change to `expected_code=[201]` since `expected_code` is an array.

Reply:
Okay yeah sorry